zzzz_modular_occulus/code/modules/clothing/gloves/stungloves.dm

```diff
@@ -0,0 +1,128 @@
+/obj/item/clothing/gloves/stunglovemakeshift //STUNGLOVES RETURN! MUAAHAHAHAHAHAHA!
```

Comment:
Why's this entire thing copypasted instead of being a child object of the parent power gloves but with different stats!

Reply:
> There's a lotta copypasta in this! Are you sure you can't reduce it?
> 
Short answer: not really. The existing stungloves are not set up to have chilsren types. I originally tried to make to make it a child for it, but due to the way they handle sprites, it wouldn’t work.
My only options were to either change the core code file, which I prefer to avoid as much as possible to avoid future merge conflicts, or copy and paste the code for a new filepath of stungloves.
The stunglovesmaint /should/ be able to accept children types, I did adjust enough in the code for that.
> Also the onmob sprites have basically no indication that they're modified at all, so they'd be really, _really_ stealthy unless you took the time to examine someone. I know it's a pain in the ass to do it, but maybe you could redden the onmob sprites a bit so it looks like it's got wires dangling all over it or something!

I suppose I can do that! I kind of thought the semistealthy part was a bit of the appeal, hence why the stungloves are simply called modified insulated gloves, rather then makeshift stungloves.

Reply:
You can overwrite the update_icon() proc in the child file however. Most of this doesn't need to be copy-pasted.